src/button/ux-button.ts

```diff
@@ -29,12 +29,24 @@ export class UxButton implements Themable {
     if (this.theme) {
       this.styleEngine.applyTheme(this, this.theme);
     }
+
+    if (this.disabled || this.disabled === '') {

```

Comment:
@ZHollingshead, hey wanted to run this possible change past ya and get feedback.  I've casted the disabled variable per @RichiCoder1's suggestion:

```
   @bindable public disabled: boolean | string = false;
```

Now I'm working on the subsequent if block to clear that logic up a bit.  Specifically, I wanted to highlight that we are attempting to detect an empty string as true.  All other casting from string to boolean will behave as normal. As well as all other castings to boolean will behave as normal.

```
    // ensure we cast empty string as true
    if (typeof this.disabled === 'string' && this.disabled === '') {
      this.disabled = true;
    }

    if (this.disabled) {
      this.button.setAttribute('disabled', '');
    }
```

Also wondering if we should be using `this.button.setAttribute('disabled', 'true');` instead like we've done in the `disabledChanged` hander?